### DISCUSSION TITLE
Wsparcie dla JSON5

### DISCUSSION THREAD

**Discussion by specpc** [hackerspace-silesia/Smogomierz#125] [Ideas]
Witam,

czy planujecie może wsparcie dla JSON5? Niestety wiele aplikacji ludzi wykorzystuje JSON5, natomiast aktualny kod jest e JSON6. 

Pozdrawiam
Matt

**Comment by bfaliszek** [comment]
Oficjalnie ArduinoJSON6 weszło pod koniec lutego 2019, to ponad rok temu. Nie planuję wprowadzać takiej kompatybilności wstecznej. Jeśli ktoś używa JSON5 to powinien pomyśleć raczej o aktualizacji do wersji 6.
